Rebrand from EventStore to Kurrent (#828)
EventStore/documentation: docs/getting-started/quickstart/README.md

In this quickstart, you will:

> - Start an KurrentDB server using Docker in GitHub Codespaces.

Reviewer: ```suggestion
- Start a KurrentDB server using Docker in GitHub Codespaces.
```

   Alternatively, you can locate and open the file from the EXPLORER window found on the left side of Codespaces. You can find the path to the file in the command above.
   :::

2. In step 3 of the script, review how KurrentDB is started with the `docker run` command:

```bash
docker run                        # Start a new Docker container using the 'docker run' command
     -d \                         # Run the container in detached mode (in the background)
     --name esdb-node \           # Assign the container a name ('esdb-node' in this case)
     -p 2113:2113 \               # Map port 2113 on the host to port 2113 in the Docker container. Required for the KurrentDB

>      kurrent-lts/kurrentdb:lts \  # Specify the Docker image to use, in this case, the KurrentDB long-term support version (lts)

Reviewer: See the previous suggestion about docker:
```suggestion
     kurrentplatform/kurrentdb:latest \  # Specify the Docker image to use, in this case, the latest supported version of KurrentDB
```

Alternatively, you can locate and open the file from the EXPLORER window on the left of Codespaces. You can find the path to the file in the command above.
:::

2. In step 1 of the code, review how the client connects to KurrentDB:

::: tabs#dev-language-platform
@tab Select >

@tab Python
```python
# Create an instance of EventStoreDBClient, connecting to the KurrentDB at localhost without TLS

> client = EventStoreDBClient(uri="esdb://localhost:2113?tls=false")  

Reviewer: The connection string will need to be updated for all of these:
```suggestion
client = EventStoreDBClient(uri="kurrentdb://localhost:2113?tls=false")  
```
And is `EventStoreDBClient` correct here, or has that been renamed to `KurrentDBClient`?

Author: Ideally code snippets should not be hardcoded here. Since this is the only file, I'll adjust them directly.

Alternatively, you can locate and open the file from the EXPLORER window on the left of Codespaces. You can find the path to the file in the command above.
:::

2. In step 1 of the code, review how the client connects to KurrentDB:

::: tabs#dev-language-platform
@tab Select >

@tab Python
```python
# Create an instance of KurrentDBClient, connecting to the KurrentDB at localhost without TLS
client = KurrentDBClient(uri="kurrentdb://localhost:2113?tls=false")  
```

@tab Java
```java
// configure the settings to connect to KurrentDB locally without TLS
KurrentDBClientSettings settings = KurrentDBConnectionString.parseOrThrow("kurrentdb://localhost:2113?tls=false");
// apply the settings and create an instance of the client
KurrentDBClient client = KurrentDBClient.create(settings);
```

@tab .NET
```c#
// Create an instance of EventStoreClientSettings, connecting to the KurrentDB at localhost without TLS

> var settings = EventStoreClientSettings.Create("kurrentdb://localhost:2113?tls=false");

Reviewer: Just to confirm - is `EventStoreClientSettings` intentional here (and further down in this file?) I've not used the rebranded clients yet, so I don't know if this has changed or not

Author: Yes! .net client haven't been rebranded yet. This will come later

:::

2. In step 1 of the code, review how the client connects to KurrentDB:

::: tabs#dev-language-platform
@tab Select >

@tab Python
```python
# Create an instance of KurrentDBClient, connecting to the KurrentDB at localhost without TLS
client = KurrentDBClient(uri="kurrentdb://localhost:2113?tls=false")  
```

@tab Java
```java
// configure the settings to connect to KurrentDB locally without TLS
KurrentDBClientSettings settings = EventStoreDBConnectionString.
   parseOrThrow("kurrentdb://localhost:2113?tls=false");

// apply the settings and create an instance of the client
KurrentDBClient client = KurrentDBClient.create(settings); 
```

@tab .NET
```c#
// Create an instance of EventStoreClientSettings, connecting to the KurrentDB at localhost without TLS

> var settings = EventStoreClientSettings.Create("kurrentdb://localhost:2113?tls=false");

Reviewer: Same as above
